default.properties

```diff
@@ -37,6 +37,16 @@ useLegacyHeader=false
 # default /header/
 headerUrl=/header/
 
+# Stylesheet used to override default colors of header
+# More design can be set by overriding default classes & styles
+# Default empty string
+#georchestraStylesheet=http://localhost:3000/tmp.css
```

Comment:
Not of great importance, but `style.css` would probably be more explicit naming.